docs/cockpit.rst

```diff
@@ -0,0 +1,107 @@
+Cockpit on Atomic Host
+======================
+
+`Cockpit <http://cockpit-project.org/>`_ is a **remote manager for GNU/Linux servers**.
+
+- Cockpit is a server manager that makes it easy to administer your GNU/Linux servers via a web browser.
+- Cockpit makes it easy for any sysadmin to perform simple tasks, such as administering storage, inspecting journals and starting and stopping services.
```

Comment:
It might be helpful to note here that you can also perform Docker related tasks from the Cockpit UI